network/src/counters.rs

```diff
@@ -156,9 +156,6 @@ pub static PENDING_RPC_NOTIFICATIONS: &str = "pending_rpc_notifications";
 /// Counter of pending requests for each remote peer
 pub static PENDING_PEER_REQUESTS: &str = "pending_peer_requests";
 
-/// Counter of pending outbound messages in Direct Send for each remote peer
-pub static PENDING_DIRECT_SEND_OUTBOUND_MESSAGES: &str = "pending_direct_send_outbound_messages";
-
```

Comment:
that metric is already captured in `PENDING_DIRECT_SEND_MESSAGES`. The metric that was removed was earlier useful because we had queuing inside DirectSend module was well. Since we've removed the queuing now, it made sense to get rid of the metric as well